Sources/Harmony/Common/MemoryReporter.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright 2018 Mobile Jazz SL
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import MachO
+
+
+/// A description of the usage of the RAM memory
+public struct RamState: CustomStringConvertible {
+    
+    let total: UInt64 // In bytes
+    let free: UInt64  // In bytes
+    let used: UInt64  // In bytes
```

Comment:
@newboadki attributes are not public. If not public, they won't be accessible from outside the library.